src/main/java/org/ow2/proactive/procci/model/occi/metamodel/MixinBuilder.java

```diff
@@ -172,4 +159,12 @@ public Mixin buildMock() {
         return build();
     }
 
+    protected String convertAttributeInString(Object attribute) throws SyntaxException {
+        try {
+            return (String) attribute;
+        } catch (ClassCastException e) {
+            throw new SyntaxException(attribute.toString());
```

Comment:
rename the method : convertAttributeAsString 
It is super weird to catch a ClassCastException ! You can eventually use instanceof . 

Additionally, if the attribute has a wrong type, you will not see in the Exception thrown which attribute key generated the error.

This is why I would suggest to write rather a method like:
readAttributeAsString(Map attributes, String key) which would be able to throw an understandable error message.